backend/spec/export_ead3_spec.rb

```diff
@@ -1377,7 +1383,7 @@ def description_content(obj)
   describe "Test unpublished record EAD exports" do
 
     def get_xml_doc(include_unpublished = false)
-      doc_for_unpublished_resource = get_xml("/repositories/#{$repo_id}/resource_descriptions/#{@unpublished_resource_jsonmodel.id}.xml?include_unpublished=#{include_unpublished}&include_daos=true&ead3=true", true)
+      doc_for_unpublished_resource = get_xml("/repositories/#{$repo_id}/resource_descriptions/#{@unpublished_resource_jsonmodel.id}.xml?include_unpublished=#{include_unpublished}&include_daos=true&include_uri_unitids=true&ead3=true", true)
```

Comment:
Same thing here, we should ensure that the default behavior happens if no `include_uri_unitids` param is given.